nameifier.go

```diff
@@ -14,6 +13,11 @@ const (
 	AdjectivesFile = "data/adjectives.json"
```

Comment:
Can we remove these if there aren't used any more?

Reply:
yep